### PR TITLE
sequel: add gem to currency reports

### PR DESCRIPTION
**Pull request by arjun-rajappa**
Adding sequel gem to currency reports.

### CODE REVIEW

File: .tekton/.currency/resources/table.json [hunk 1]
```diff
@@ -89,6 +89,12 @@
         "Support Policy": "45-days",
         "Beta version": "No",
         "Cloud Native": "Yes"
+      },
+      {
+        "Package name": "Sequel",
+        "Support Policy": "45-days",
```
Comment (pvital) [inline .tekton/.currency/resources/table.json:95]: The support policy is listed here as 45 days, but in the table of the `.tekton/.currency/docs/report.md` file, it's listed as 30 days. Something is not updated :-D

Comment (arjun-rajappa) [reply .tekton/.currency/resources/table.json:95]: This .tekton/.currency/docs/report.md is just a temparory file i guess. When generating the report we exclusively use [.tekton/.currency/resources/table.json](https://github.com/instana/ruby-sensor/pull/423/files/f1a84bd976cee103554d02c2c4a4234ed05ce4de#diff-7aca6c9eaea4eb00b8f9038a883aeaf0ddc6ec03d0917d71491a566d5f6076da) . If we look at the reports generated and uploaded - https://github.ibm.com/instana/tracer-reports/blob/main/automated/currency/ruby/report.md . It says 45 days.

Comment (GSVarsha) [reply .tekton/.currency/resources/table.json:95]: @arjun-rajappa , yes this an example file and is subjected to change but would be good to have the most updated report structure especially when fields like `Support Policy` won't change much.

Comment (arjun-rajappa) [reply .tekton/.currency/resources/table.json:95]: @GSVarsha and @pvital  I have updated the report to the latest report!

**Review by pvital — APPROVED** [review]
It looks good to me.

**Pull request merged**